librosa/core/constantq.py

```diff
@@ -565,4 +573,3 @@ def __num_two_factors(x):
         x //= 2
 
     return num_twos
```

Comment:
I feel like this could be made much more efficient by counting the trailing zeros in the binary representation.  Maybe not necessary though.

At any rate, there should be an initial cast to int.